example/docker-compose/tempo-search/docker-compose.yaml

```diff
@@ -40,8 +40,9 @@ services:
       - "9090:9090"
 
   grafana:
-    image: grafana/grafana:8.0.3
+    image: grafana/grafana:main   # track main as search is under active development
```

Comment:
I'm not a huge fan of tracking `main`, but I think this is acceptable until search is part of a release (which will be 8.2 probably)